tests/PHPStan/Analyser/nsrt/bug-9435.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bug9435;
+
+use function PHPStan\Testing\assertType;
+
+function x(): mixed
+{
+	return null;
+}
+
+/** @phpstan-assert ($allow_null is true ? string|null : string) $input */
+function trueCheck(mixed $input, bool $allow_null = false): void
```

Comment:
Maybe test a function expecting variadics

Reply:
What do you mean ? Could you give a phpstan.org snippet ?